PaTTAN Room Displays/PaTTAN Room Displays/MainPage.xaml

```diff
@@ -1,12 +1,13 @@
 <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-             x:Class="PaTTAN_Room_Displays.MainPage">
+             x:Class="PaTTAN_Room_Displays.MainPage"
+             BackgroundColor=" #b8daff"
+             >
 
     <StackLayout>
         <Grid RowSpacing="25" 
```

Comment:
This is meant to just display the next event for the room, not all events for the day?

Reply:
Yes.  I was thinking we would check every 5 min for conference room data.  We show only the current meeting or the next one coming up.   If all meetings are done, we display 'no scheduled meetings' or something to that effect.